calculate-info.patch

```diff
@@ -0,0 +1,30 @@
+diff --git a/activity/activity.info b/activity/activity.info
+index cc53226..8bf18db 100644
+--- a/activity/activity.info
++++ b/activity/activity.info
+@@ -1,12 +1,19 @@
+ [Activity]
+ name = Calculate
+-bundle_id = org.laptop.Calculate
++bundle_id = org.sugarlabs.Calculate
+ icon = calculate
+ exec = sugar-activity3 calculate.Calculate -s
+ activity_version = 47
+-license = GPLv2+
```

Comment:
Keep the license please, see https://github.com/sugarlabs/calculate-activity/blob/master/COPYING